packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/ColorUtilsTest.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+import android.graphics.Color
+import androidx.annotation.Size
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+internal class ColorUtilsTest {
+
+    @Test
+    fun `M use default alpha W resolveAlphaValue { color without opacity }`() {
+        // When
+        val alphaValue = resolveAlphaValue(0.1f, 0x000000)
+
+        // Then
+        assertThat(alphaValue)
+            .isEqualTo(25)
+    }
+
+    @Test
+    fun `M use opacity code W resolveAlphaValue { color with opacity }`() {
+        // Given
+        val color = 1717960806
+
+        // When
+        val alphaValue = resolveAlphaValue(0.1f, color)
+
+        // Then
+        assertThat(alphaValue)
+            .isEqualTo(102)
```

Comment:
Could you explain how `102` is the expected value? You can translate from decimal to hex in comments for more clarity